app/templates/spring-boot/README.md

```diff
@@ -0,0 +1,19 @@
+<%= userProps.appName %>
```

Comment:
in the generated readme I have a blank first line, seems the value is not replaced (or we miss a parameter to configure it?)

![image](https://user-images.githubusercontent.com/1105127/50440648-7981ba00-08f7-11e9-8013-eba23aa3fc28.png)

Reply:
Interesting. I'll see if I can figure out what's going on there.

Reply:
Figured out this issue. Should be userProps.name instead of userProps.appName. Will also address this in the other readme files.